packages/service-library/src/servicelib/redis_utils.py

```diff
@@ -70,6 +70,7 @@ def start_exclusive_periodic_task(
     task: Callable[..., Awaitable[None]],
     *,
     interval: timedelta,
+    exclusive_task_starter_interval: timedelta = timedelta(seconds=1),
```

Comment:
if I may, I find the naming is very confusing. 
Suggestion:
rename ```interval``` in ```task_period``` or ```task_interval```
and maybe copying tenacity, ```exlusive_task_starter_interval``` into ```retry_after``` or similar

Reply:
Thanks for the tip, it sounds way better now.